live-examples/css-examples/scroll-snap/meta.json

```diff
@@ -1,11 +1,158 @@
-            "exampleCode": "./live-examples/css-examples/scroll-snap/scroll-snap-type.html",
-            "fileName": "scroll-snap-type.html",
-            "title": "CSS Demo: scroll-snap-type",
-            "type": "css"
-        }
-    }
-}
+{
+    "pages": {
+        "scrollMargin": {
+            "cssExampleSrc": "./live-examples/css-examples/scroll-snap/scroll-margin.css",
+            "exampleCode": "./live-examples/css-examples/scroll-snap/scroll-margin.html",
+            "fileName": "scroll-margin.html",
+            "title": "CSS Demo: scroll-margin",
+            "type": "css"
+        },
+        "scrollMarginBlock": {
+            "cssExampleSrc": "./live-examples/css-examples/scroll-snap/scroll-margin-top.css",
+            "exampleCode": "./live-examples/css-examples/scroll-snap/scroll-margin-block.html",
+            "fileName": "scroll-margin-block.html",
+            "title": "CSS Demo: scroll-margin-block",
+            "type": "css"
+        },
+        "scrollMarginBlockEnd": {
+            "cssExampleSrc": "./live-examples/css-examples/scroll-snap/scroll-margin-bottom.css",
+            "exampleCode": "./live-examples/css-examples/scroll-snap/scroll-margin-block-end.html",
+            "fileName": "scroll-margin-block-end.html",
+            "title": "CSS Demo: scroll-margin-block-end",
+            "type": "css"
+        },
+        "scrollMarginBlockStart": {
+            "cssExampleSrc": "./live-examples/css-examples/scroll-snap/scroll-margin-top.css",
+            "exampleCode": "./live-examples/css-examples/scroll-snap/scroll-margin-block-start.html",
+            "fileName": "scroll-margin-block-start.html",
+            "title": "CSS Demo: scroll-margin-block-start",
+            "type": "css"
+        },
+        "scrollMarginBottom": {
+            "cssExampleSrc": "./live-examples/css-examples/scroll-snap/scroll-margin-bottom.css",
+            "exampleCode": "./live-examples/css-examples/scroll-snap/scroll-margin-bottom.html",
+            "fileName": "scroll-margin-bottom.html",
+            "title": "CSS Demo: scroll-margin-bottom",
+            "type": "css"
+        },
+        "scrollMarginInlineEnd": {
+            "cssExampleSrc": "./live-examples/css-examples/scroll-snap/scroll-margin-right.css",
+            "exampleCode": "./live-examples/css-examples/scroll-snap/scroll-margin-inline-end.html",
+            "fileName": "scroll-margin-inline-end.html",
+            "title": "CSS Demo: scroll-margin-inline-end",
+            "type": "css"
+        },
+        "scrollMarginInlineStart": {
+            "cssExampleSrc": "./live-examples/css-examples/scroll-snap/scroll-margin.css",
+            "exampleCode": "./live-examples/css-examples/scroll-snap/scroll-margin-inline-start.html",
+            "fileName": "scroll-margin-inline-start.html",
+            "title": "CSS Demo: scroll-margin-inline-start",
+            "type": "css"
+        },
+        "scrollMarginInlineRight": {
```

Comment:
I think this should be `scrollMarginRight`, shouldn't it?

Reply:
oops yes! Fixed.